src/components/fields/schemaFields/widgets/varPopup/VarMenu.tsx

```diff
@@ -212,6 +215,12 @@ const VarMenu: React.FunctionComponent<VarMenuProps> = ({
     onSelect: onVarSelect,
   });
 
+  // Render a hidden element if the component has not been positioned yet to avoid jumpiness
+  // when the menu is shown. This is needed because the position is computed asynchronously (see usePositionVarPopup).
+  // Also see the discussion thread for floating-ui to support synchronous computePosition:
+  // https://github.com/floating-ui/floating-ui/discussions/2720
+  if (!positioned) return <div className="hidden" ref={rootElementRef} />;
```

Comment:
You can use the native `hidden` attribute

```suggestion
  if (!positioned) return <div hidden ref={rootElementRef} />;
```

Reply:
Unfortunately using the hidden attribute doesn't work in this case. I didn't spend a lot of time looking into it but it may likely be due to the `display` css property being overriden.

https://css-tricks.com/the-hidden-attribute-is-visibly-weak/

Reply:
I'm in the `hidden` attr shouldn't have made it into the spec camp. Presentational concerns should belong to css, not html.